google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -757,11 +755,10 @@ public String getAppProfileId() {
      * Sets if channels will gracefully refresh connections to Cloud Bigtable service.
      *
      * <p>When enabled, this will wait for the connection to complete the SSL handshake and warm up
-     * serverside caches for all the tables of the instance.
+     * serverside caches for all the tables of the instance. This feature is enabled by default.
      *
      * @see com.google.cloud.bigtable.data.v2.BigtableDataSettings.Builder#setRefreshingChannel
      */
```

Comment:
Lets deprecate this and say that this will eventually be removed

Reply:
Done!